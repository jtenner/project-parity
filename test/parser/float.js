const { OptionType } = require("../../lib");

module.exports = {
  argv: ["--option-a", "3.14"],
  config: {
    optionA: {
      type: OptionType.Float,
      default: 42,
    },
  },
};
