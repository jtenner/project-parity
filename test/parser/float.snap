{
  args: [ 'test\\parser\\float.config.js' ],
  cliProvided: Set(1) { 'optionA' },
  configProvided: Set(0) {},
  errors: [],
  options: { optionA: 3.14 },
  unknown: [ '--config' ]
}
